packages/web-react/tests/providerTests/requiredPropsTest.tsx

```diff
@@ -0,0 +1,18 @@
+import { render, screen } from '@testing-library/react';
+import React, { ComponentType } from 'react';
+
+export const requiredPropsTest = (
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  Component: ComponentType<any>,
```

Comment:
```suggestion
  Component: ComponentType<unknown>,
```

Please, try to use `unknow` instead of `any`. It is somehow better. Or maybe we should pass the type of the component as a generic or maybe the `typeof Component` can be working.

Reply:
Unfortunately, I couldn't type it either on unknown or on generic type.. however, all other providerTests also use "any" type